neuralg/io/download_models.py

```diff
@@ -0,0 +1,27 @@
+from loguru import logger
+import requests
+import zipfile
+import io
+import os
+
+
+def download_models():
+    """Downloads and unzips the folder of model state dicts attached to the latest GitHub release.
+    Will save the folder as saved_models in the models subdirectory.
+    """
+    logger.info("Downloading models from the latest release")
+    # This is currently pre-release version
+    zip_url = "https://github.com/esa/neuralg/releases/download/v.0.0.1-alpha/saved_models.zip"
```

Comment:
if you set a [__version__ string](https://peps.python.org/pep-0008/#module-level-dunder-names) for the module you could possible directly access that here 🤔

Reply:
(to avoid having to manually having to change it)

did you check, if you can replace the file if necessary? otherwise, it may get annoying because it will require you to create new release / pip version whenever you want to change the models

Reply:
Do you mean for the logger or in the actual link? :) I think for the link it should work with the /latest/download/... thing , as long as the release is marked as the latest.


Yes it works when replacing the file! 👍

Reply:
Something that I find a little bit annoying is the indexing for testpypi versus release and pypi index (I guess you want the latter to be the same ideally?) cause I manually have to change between then? Or am I just doing it stupid?